devdocs/ReactIntegration.md

```diff
@@ -17,7 +17,7 @@ Using JsInterop to call your React component can be summarized in these basic st
 
 1. Create a class to represent your components' props
 1. Add your component to the SRC JsInterop type
-1. Add a ReactComponentDiv to your View
+1. Add a ReactComponent to your View
```

Comment:
Renamed `ReactComponentDiv` to `ReactComponent`. `ReactComponentSpan` was removed; the root element of the `ReactComponent` can be chosen when calling the widget constructor.